src/arevtur/xElements/itemListing/ItemListing.html

```diff
@@ -112,6 +112,7 @@
 	<div>
 		<span class="value">Value: <span id="value-text"></span></span>
 		<span id="value-build-link" class="hidden">(build)<span id="value-build-tooltip"></span></span>
+		<button id="copy-item-button">Copy to Clipboard</button>
```

Comment:
```suggestion
		<button id="copy-item-button">Copy item</button>
```
Just to make it clear what's being copied.